### PR TITLE
Add RESOURCES page to track community material

### PR DESCRIPTION
**Pull request by samueltauil**
Signed-off-by: Samuel Tauil <samueltauil@gmail.com>

Possible and initial solution to the discussion: https://github.com/open-gitops/project/discussions/99

### CODE REVIEW

File: RESOURCES.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+5. **Case Studies**
+   - Real-world examples and case studies showcasing organizations' experiences and benefits of implementing GitOps practices in different environments.
+
+6. **Videos and Webinars**
+   - Recorded talks, presentations, or webinars discussing GitOps concepts, tool demonstrations, case studies, and expert interviews.
+
+7. **Tutorials and Guides**
+    - Step-by-step tutorials, guides, or manuals providing instructions on how to implement specific GitOps practices, set up tools, or solve common challenges.
+
+8. **Courses and Training Materials**
```
Comment (scottrigby) [inline RESOURCES.md:28]: i think links to each section would help. WDYT?

```suggestion
1. [Books](#books)
   - Comprehensive guides or textbooks covering GitOps principles, practices, and implementations.

2. [Research Papers](#research-papers)
   - Academic papers exploring various aspects of GitOps, including methodologies, comparisons with other DevOps practices, and empirical research.

3. [Articles](#articles)
   - Online or print articles discussing specific aspects of GitOps, such as tutorials, best practices, tool comparisons, and industry trends.

4. [Documentation](#documentation)
   - Official or community-contributed documentation for GitOps tools, platforms, and frameworks, providing guidance on installation, configuration, and usage.

5. [Case Studies](#case-studies)
   - Real-world examples and case studies showcasing organizations' experiences and benefits of implementing GitOps practices in different environments.

6. [Videos and Webinars](#videos-and-webinars)
   - Recorded talks, presentations, or webinars discussing GitOps concepts, tool demonstrations, case studies, and expert interviews.

7. [Tutorials and Guides](#tutorials-and-guides)
    - Step-by-step tutorials, guides, or manuals providing instructions on how to implement specific GitOps practices, set up tools, or solve common challenges.

8. [Courses and Training Materials](#courses-and-training-materials)
```

**Review by scottrigby — COMMENTED** [review]
@samueltauil Nice tie into the older discussion about fact-checking team and community resources page. Just one formatting suggestion.

* note this PR was split out from https://github.com/open-gitops/documents/pull/78

**Comment by samueltauil** [comment]
great suggestion! changes applied @scottrigby

**Review by scottrigby — APPROVED** [review]
This is a great start @samueltauil 👏 I know a lot of folks are looking forward to adding material here, thanks for getting a structure for this community resources page in place!

**Pull request merged**